modules/monitoring-creating-a-monitoringstack-object-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-monitoringstack-object-for-cluster-observability-operator_{context}"]
+= Creating a MonitoringStack object for Cluster Observability Operator
+
+To be able to discover the sample `ServiceMonitor` resource you created, you must create and configure a `MonitoringStack` object.
+
+This procedure shows how to create a `MonitoringStack` object to discover the sample `ServiceMonitor` resource for a sample service named `prometheus-coo-example-app` in the `ns1-coo` namespace.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role or as a user with administrative permissions for the namespace.
+* You have installed the Cluster Observability Operator.
```

Comment:
recommend adding an attribute for the Operator name

Reply:
Agree. Will do.